goodbye.rb

```diff
@@ -0,0 +1,3 @@
+puts "Hello!. what is your name?"
+answer = gets.chomp
+puts "It was a pleaseure #{answer}. I shall see you soon"
```

Comment:
Please, Add new line!

Reply:
Thanks! I will keep it in mind next time